cpp/src/gandiva/function_registry_common.h

```diff
@@ -285,4 +285,13 @@ typedef std::unordered_map<const FunctionSignature*, const NativeFunction*, KeyH
 #define NUMERIC_BOOL_DATE_VAR_LEN_TYPES(INNER, NAME, ALIASES) \
   NUMERIC_BOOL_DATE_TYPES(INNER, NAME, ALIASES), VAR_LEN_TYPES(INNER, NAME, ALIASES)
 
+// Is True and Is Not True functions with validity that :
+// - NULL handling is of type NULL_NEVER
+//
+// The pre-compiled fn name includes the base name & input type name. istrue_boolean /
+// isfalse_boolean
+#define ISTRUE_ISFALSE_NULL_NEVER(NAME, ALIASES, IN_TYPE, OUT_TYPE)                  \
```

Comment:
You can follow the example of `UNARY_SAFE_NULL_IF_NULL` and create a more generic macro, like `UNARY_SAFE_NULL_NEVER`, if it does not exist yet